### PR TITLE
Test jenci

### PR DESCRIPTION
**Pull request by jenci2114**
Added 4 tests

### CODE REVIEW

File: src/test/java/EventGetterTest.java [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+import services.event_creation.EventInfoFromReader;
+import services.event_presentation.CalendarEventPresenter;
+import services.event_presentation.EventGetter;
+import services.event_presentation.EventInfo;
+
+import java.io.IOException;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+import java.util.*;
```
Comment (ckatsube) [inline src/test/java/EventGetterTest.java:16]: Change your IDE settings to not bundle imports into the wildcard (you should be able to set the count to like 999)

File: src/test/java/TaskOutputterTest.java [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+
+        }
+
+        @Override
+        public void saveTodo(String filepath) throws IOException {
+
+        }
+    }
+
+    private class MockTodoListPresenter implements TodoListPresenter {
```
Comment (ckatsube) [inline src/test/java/TaskOutputterTest.java:79]: Make sure to resolve warnings like making this class static, or not having unused exception declarations (in the two methods above)

File: src/main/java/services/event_creation/EventInfoFromReader.java [hunk 3]
```diff
@@ -49,4 +50,17 @@ public LocalTime getEndTime() {
     public boolean getCompleted() {
         return eventReader.getCompleted();
     }
+
+    @Override
+    public boolean equals(Object o) {
```
Comment (ckatsube) [inline src/main/java/services/event_creation/EventInfoFromReader.java:55]: why are adding methods purely for the test suite? or was there some larger intention when including this.
Also if you're going to override equals, might as well make it property-based, allowing for equality across other EventInfo instead of a wrapper based comparison with another concrete EventInfoFromReader. The current method exposes the underlying implementation underneath the abstraction, making it so that client code has to be aware that the two items it's comparing are EventInfoFromReader in order to compare equality (it should be something more abstract like the properties are the same)

File: src/main/java/services/task_presentation/TodoListInfoFromTaskReaders.java [hunk 4]
```diff
@@ -31,4 +31,8 @@ public TodoListInfoFromTaskReaders(Map<Long, List<TaskReader>> taskReaders) {
             taskInfos.add(new TaskInfoFromTaskReader(tr));
         return taskInfos;
     }
+
+    public List<TaskReader> getTaskReaders() {
```
Comment (ckatsube) [inline src/main/java/services/task_presentation/TodoListInfoFromTaskReaders.java:35]: If you want to expose the Readers to the adapter and outward layers, might as well deprecate the -Info classes, but make that kind of change another PR. I think I was going to eventually go about doing this with the observable repositories PR but I hadn't committed to the idea yet

File: src/test/java/EventGetterTest.java (hunk 1 above)
Comment (jenci2114) [reply src/test/java/EventGetterTest.java]: fixed

File: src/test/java/TaskOutputterTest.java (hunk 2 above)
Comment (jenci2114) [reply src/test/java/TaskOutputterTest.java]: OK

**Review by ckatsube — APPROVED** [review]

File: src/main/java/services/event_creation/EventInfoFromReader.java (hunk 3 above)
Comment (jenci2114) [reply src/main/java/services/event_creation/EventInfoFromReader.java]: OK I deleted the equals method

File: src/main/java/services/task_presentation/TodoListInfoFromTaskReaders.java (hunk 4 above)
Comment (jenci2114) [reply src/main/java/services/task_presentation/TodoListInfoFromTaskReaders.java]: Ok I removed the getter
